{
  callBack.runTestMethod(testResult);
  Throwable testResultException=testResult.getThrowable();
  if (testResultException instanceof InvocationTargetException) {
    testResultException=((InvocationTargetException)testResultException).getCause();
  }
  this.testException=testResultException;
}
